### DISCUSSION TITLE
How can I prevent autoplay in safari/iOS?

### DISCUSSION THREAD

**Discussion by randyriback** [gilbarbara/react-spotify-web-playback#83] [Q&A] [answered]
I'm basically hoping to achieve something similar to [this in the SDK](https://developer.spotify.com/documentation/web-playback-sdk/reference/#api-spotify-player-activateelement) but am having trouble.

My Spotify Player component spins up with an onClick that set's the play status, but also fetches the data from the Spotify Web API. 

Are there any work-arounds you can think of similar to player.activateElement() within '
react-spotify-web-playback'?

**Comment by gilbarbara** [comment]
hey @randyriback 

I didn't get your question...
You can control it with the `autoPlay` prop and it works for all platforms. Is this not working for you?

[s]

**Reply by randyriback** [reply]
I'm sorry if it wasn't clear-- I'm still new at developing.

Yes, I tried the autolplay prop... this is a noobie question but when setting the boolean, would I do "autoplay={false}" or autoplay="false" ?

thanks so much for your time and lovely component!

**Comment by gilbarbara** [comment, marked as answer]
Ahh, ok.

It's a boolean, so `autoPlay={false}`. But it's `false` by default.

Anyway, everything inside the curly brackets `{}` is evaluated, while everything with quotes it'll always be a string.

**Comment by randyriback** [comment]
I see, I'd been triggering the `play` prop with a useEffect and it was overriding the `autoPlay`... thanks so much for your help. This Spotify Player is a lot of fun.
